### DISCUSSION TITLE
Admin API - generic logout for linked session by parameter

### DISCUSSION THREAD

**Discussion by JohnGreaves** [ory/kratos#1083] [Q&A] [answered]
Hello, my team has questions. 
1. Are 'Kratos' has API for SSO logout for multiply sessions which linked may be by email or something like this?
2. Can we make the session cookie lifetime the same as the lifetime of the access token (For example access token which provides Google Identity provider)?

**Comment by zepatrik** [comment, marked as answer]
1. Maybe https://www.ory.sh/kratos/docs/reference/api#revoke-and-invalidate-a-session is enough for you; otherwise see https://github.com/ory/kratos/issues/655
2. Search for `lifespan` or `session` here: https://www.ory.sh/kratos/docs/reference/configuration
